packages/directives/click-outside/index.ts

```diff
@@ -71,7 +71,7 @@ function createDocumentHandler(
     ) {
       return
     }
-    binding.value()
+    binding.value(mousedown)
```

Comment:
maybe we should expose both event to the handler so that they can do comparison to the `mousedown` target and `mouseup` target.

Also make sure your code is tested.